angular/projects/spark-angular/src/lib/directives/sprk-button/sprk-button.stories.ts

```diff
@@ -24,6 +25,7 @@ for accessibility to choose the correct element.
 an action, such as: “Submit”, “Add”, “Join”,” etc.
 - A Button that does not go to a new page will almost
 always use a \`<button>\` element.
+- Spinner requires additional engineering. Use app state to conditionally render button text and spinner. (For example, use \`ngIf\` to to switch to spinner.)
```

Comment:
There is a double `to to` in here. I think what you wrote makes sense to me but might not to others. Trying to think of what we would write here that might be less wordy, "app state" might not be something angular devs commonly refer too.

Reply:
✅